demos/src/css/chartAuthoring-demo.css

```diff
@@ -0,0 +1,25 @@
+.floe-ca-pieChart-pie {
+    float: left;
+    width: 400px;
```

Comment:
At the moment this would only scale the container size - if we want the drawn pie chart to be responsive to element text size changes, I can work on implementing https://issues.fluidproject.org/browse/FLOE-414 (I have a basic autoscale working that will set the pie chart's size at create time based on the DIV container size)